core/arch/arm/mm/tee_pager.c

```diff
@@ -1027,8 +1027,6 @@ static void rem_region(struct vm_paged_region_head *regions,
 	}
 
 	pager_unlock(exceptions);
-
-	free_region(reg);
 }
 DECLARE_KEEP_PAGER(rem_region);
```

Comment:
I wonder why rem_region() needs to be unpaged since its called only from tee_pager_rem_um_region() which has no pager constraints.

Reply:
Because it holds the pager lock.

Reply:
indeed, right!